### DISCUSSION TITLE
What are the Secrets for alerts like PROCESSOR_ICON or WARNING_COLOR

### DISCUSSION THREAD

**Discussion by Headkillah** [jazzyisj/home-assistant-config#79] [General]
Hello jazzyisj, it´s me again :)

First thx for all the updates :)

In your alert configurations (like package_system.yaml) you have entries like "icon_url: !secret PROCESSOR_ICON" or "color: !secret WARNING_COLOR".

I absolutely do not understand how the structure for this is :(

Can you post an example or something else?

Thx in advance

**Comment by jazzyisj** [comment]
I use the secrets.yaml as kind of a hacky global variable declaration (as well as storing API's etc)
Here's a couple snippets from my secrets file.  You don't need to use a secret, simply supplying a valid value will work.

```
HASS_COLOR: "#4B9BEF"
CRITICAL_COLOR: "#B30000"
SEVERE_COLOR: "#FF6600"
WARNING_COLOR: "#FFAD33"
MINOR_COLOR: "#FFFF66"
NOTIFY_COLOR: "#19BFBF"

PIHOLE_ICON: /local/images/notification/pihole_icon.png
PLANTS_IMAGE: /local/images/notification/water_plants_image.png
PLANTS_ICON: /local/images/notification/water_plants_icon.png
POWER_ICON: /local/images/notification/power_use_icon.png
PROCESSOR_IMAGE: /local/images/notification/processor_image.png
PROCESSOR_ICON: /local/images/notification/processor_icon.png
```

**Reply by Headkillah** [reply]
I already thought so with the paths, thx for clearifying :) Thx again fo rthe fast answer :)
